client/cli/src/runner.rs

```diff
@@ -106,9 +106,7 @@ where
 	pin_mut!(f);
 
 	tokio_runtime.block_on(main(f)).map_err(|e| e.to_string())?;
-
-	task_manager.terminate();
-	drop(tokio_runtime);
+	task_manager.clean_shutdown();
```

Comment:
What is the difference? And why isn't `clean_shutdown` being called in `Drop`?

Reply:
No, clean_shutdown is async.... wait, I forgot the .await!!! :sweat: